### PR TITLE
1.5 for short circuit

### PR DESCRIPTION
**Pull request by TonyXiang8787**

### CODE REVIEW

**Comment by mgovers** [comment]
Apologies, i did `--no-verify` but apparently, that also bypasses the branch protection rule. that sound really bad but it is the way it is. i did a reset to the previous actual real release.

**Comment by TonyXiang8787** [comment]
@mgovers  can you approve this?

**Comment by mgovers** [comment]
> @mgovers can you approve this?

yes, but i will quickly (15min max) go over all files one last time

File: docs/user_manual/dataset-terminology.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Attributes of Components
+
+| Attribute characteristic | Description                                                                                                                                                                                                                                                                                                  |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| name                     | Name   of the attribute. It is exactly the same as the attribute name in   `power_grid_model.power_grid_meta_data`.                                                                                                                                                                                          |
```
Comment (mgovers) [inline docs/user_manual/dataset-terminology.md:60]: nitpick for follow-up PR: redundant spaces. Out of scope of this PR

File: docs/user_manual/dataset-terminology.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| required                 | Whether the attribute is required. If not, then it is optional. Note if you choose not to specify an optional attribute, it should have the null value as defined in [](../advanced_documentation/native-data-interface.md#basic-data-types).                                                                |
+| update                   | Whether the attribute can be mutated by the update call `PowerGridModel.update` on an existing instance, only applicable when this attribute is part of an input dataset.                                                                                                                                    |
+| valid values             | Whether applicable, an indication which values are valid for the input data                                                                                                                                                                                                                                  |
```
Comment (mgovers) [inline docs/user_manual/dataset-terminology.md:68]: nitpick for follow-up PR: inconsistent use of `.` at the end of the descriptions

File: src/power_grid_model/core/power_grid_model.py [hunk 3]
```diff
@@ -357,64 +379,151 @@ def calculate_state_estimation(
...
             update_data=update_data,
+            output_component_types=output_component_types,
+            options=options,
+            continue_on_batch_error=continue_on_batch_error,
+        )
+
+    def calculate_short_circuit(
+        self,
+        *,
+        calculation_method: Union[CalculationMethod, str] = CalculationMethod.iec60909,
```
[the remark this answers is not in the document]
Comment (TonyXiang8787) [reply src/power_grid_model/core/power_grid_model.py:458]: We have a design choice to not to provide the user to specify the scaling factor. It is always 1.1.

We can add this argument later if the users want.

I would keep this as is.

[the remark this answers is not in the document]
Comment (mgovers) [reply src/power_grid_model/core/power_grid_model.py:458]: The documentation includes the following lines:

> - To account for the different operational conditions, a voltage scaling factor of `c` is applied to the voltage source while running short circuit calculation function. By default, the value is considered as `1.1` in power-grid-model. 
> A detailed table for its selection is mentioned in IEC 60909.

should we update that with this design choice now or in a follow-up PR?

[the remark this answers is not in the document]
Comment (TonyXiang8787) [reply src/power_grid_model/core/power_grid_model.py:458]: I propose to merge the release as is. I have read the documentation and it is pretty clear now.

[the remark this answers is not in the document]
Comment (TonyXiang8787) [reply src/power_grid_model/core/power_grid_model.py:458]: We only provide the option to let user to select a scaling factor, if users want that.

[the remark this answers is not in the document]
Comment (mgovers) [reply src/power_grid_model/core/power_grid_model.py:458]: OK 😄 i might iterate on it when implementing https://github.com/PowerGridModel/power-grid-model/issues/330 but i agree with merging this PR as is. The documentation is clear enough for what it's worth

**Review by nitbharambe — APPROVED** [review]

**Pull request merged**